### DISCUSSION TITLE
✅ 🔗 🚀 Feature Request: Ability to add widgets with one click (Duplicate of #20142)  - [ED-10566]

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#21965] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

I remember I saw a video before, which was actually a preview of the new editor. If I am not mistaken, this video was released in 2021. In this video, this feature can be seen that widgets are added with just one click. Just like the way it is now in other page builders like bricks.
I don't know if you have any plans to add this feature, but if you do, you will solve a big problem in Elementor. Also, the speed of the workflow will be much higher.

You might ask what I mean by adding this feature you solve a big problem in Elementor. When the container is in row mode and there are widgets inside it, it is very difficult and even impossible to add a widget inside it, and we have to use the navigator. This is really a big problem.

In addition, when we add a widget, the widget settings will be opened immediately. Therefore, to add another widget, we must first go to the widgets panel and then add the desired widget. This slows down the workflow.

But when we can add widgets just by clicking on them, then we can add several widgets at once and then click on it to access its settings.

### Describe the solution you'd like

Please add the ability to add widgets with one click. It can exist alongside drag and drop. Anyone can add widgets using one click, or using drag and drop. Personally, I will quit drag and drop immediately.

**Comment by ayoubkhan558** [comment]
+1

**Comment by nicholaszein** [comment]
Hello there,

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #20142 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
